src/openrct2-ui/windows/TrackDesignManage.cpp

```diff
@@ -19,6 +19,8 @@
 static constexpr const rct_string_id WINDOW_TITLE = STR_STRING;
 static constexpr const int32_t WH = 44;
 static constexpr const int32_t WW = 250;
+static constexpr const int32_t WH2 = 74;
```

Comment:
This isn't a very descriptive name. It doesn't make it clear that this is for the track delete prompt. Also you should use these sizes for opening the window on L206. I'm somewhat surprised there isn't any drawing issues atm.

Reply:
Fixed. Also found existing code used in this file was not using the size originally defined constants, fixed that too.